cypress/e2e/homepage.cy.js

```diff
@@ -1,4 +1,4 @@
-describe('Rancid Tomatillos', () => {
+describe('Rancid Tomatillos home page', () => {
```

Comment:
Good eye on adding in more details to the describe blocks since we separated out the tests for both pages!